homeassistant/components/lutron/__init__.py

```diff
@@ -76,8 +82,12 @@ def setup(hass, base_config):
                         )
 
                 hass.data[LUTRON_BUTTONS].append(LutronButton(hass, keypad, button))
+        if area.occupancy_group is not None:
+            hass.data[LUTRON_DEVICES]["binary_sensor"].append(
+                (area.name, area.occupancy_group)
+            )
 
-    for component in ("light", "cover", "switch", "scene"):
+    for component in ("light", "cover", "switch", "scene", "binary_sensor"):
         discovery.load_platform(hass, component, DOMAIN, None, base_config)
```

Comment:
Please pass an empty dict as discovery_info, ie the fourth argument should not be `None`.